### PR TITLE
chore(wetty): update dependencies

### PR DESCRIPTION
**Pull request by frickjack**
update wetty dependencies to satisfy security

### CODE REVIEW

File: wetty/wetty.yml [hunk 1]
```diff
@@ -27,13 +37,14 @@ services:
     environment:
       - SSHHOST=localhost
       - SSHPASS=gen3
-      - SSHUSER=gen3
-      - SSHPORT=22
+      - SSHUSER=ubuntu
+      - SSHPORT=2222
```
Comment (itsJiaqi) [inline wetty/wetty.yml:35]: should this be 22?

Comment (frickjack) [reply wetty/wetty.yml:35]: it's a little confusing - this yml file is only used for local testing with docker-compose.  We actually run sshd as a non-root user in the `awshelper` container, and wire it up to listen at port 2222 ...

Comment (itsJiaqi) [reply wetty/wetty.yml:35]: oh okay, that's cool.

Comment (frickjack) [reply wetty/wetty.yml:35]: we also use a copy of this yml file when deploying wetty as a hatchery app - but that's a different use case than the "admin tty" thing ...

**Review by itsJiaqi — APPROVED** [review]
LGTM

**Pull request merged**